brainsets/utils/mne_utils.py

```
@@ -662,3 +662,37 @@ def _transpose_type_channels_mapping(
+    Returns:
+        An Interval with start/end arrays and a 'description' field, or an empty
+        Interval if no annotations are present.
+
+    Raises:
+        ImportError: If MNE is not installed.
+    """
+    _check_mne_available("extract_annotations")
+    annotations = recording_data.annotations
+
+    if len(annotations) == 0:
+        warnings.warn("No annotations found in recording data")
```

Ruff (0.15.15)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)